Nef_3/include/CGAL/Nef_3/bounded_side_3.h

```diff
@@ -70,21 +70,27 @@ Bounded_side bounded_side_3(IteratorForward first,
 
 
 
+  auto apv = approx(plane.orthogonal_vector());
+
+  int dir = 0;
```

Comment:
```c++
int dir = CGAL::abs(apv[0]) > CGAL::abs(apv[1]) ? 0 : 1;
    dir = CGAL::abs(apv[2]) > CGAL::abs(apv[dir]) ? 2 : dir;
```
Is this equivalent?

Reply:
It seems to be equivalent, but what is the goal?  Elegance, compactness, performance?

Reply:
Yeah, compactness. The performance is likely identical. it may even generate the same asm.